File: app/src/main/java/com/mifos/apache/fineract/data/models/loan/CreditWorthinessSnapshot.java
package com.mifos.apache.fineract.data.models.loan;

import java.util.List;

/**
 * @author Rajan Maurya
 *         On 12/07/17.
 */

public class CreditWorthinessSnapshot {

    private String forCustomer;
    private List<CreditWorthinessFactor> incomeSources;
    private List<CreditWorthinessFactor> assets;
    private List<CreditWorthinessFactor> debts;

    public String getForCustomer() {
        return forCustomer;
    }

    public void setForCustomer(String forCustomer) {
        this.forCustomer = forCustomer;
    }

    public List<CreditWorthinessFactor> getIncomeSources() {
        return incomeSources;
    }

    public void setIncomeSources(
            List<CreditWorthinessFactor> incomeSources) {
        this.incomeSources = incomeSources;
    }

    public List<CreditWorthinessFactor> getAssets() {
        return assets;
    }

    public void setAssets(
            List<CreditWorthinessFactor> assets) {
        this.assets = assets;
    }

    public List<CreditWorthinessFactor> getDebts() {
        return debts;
    }

    public void setDebts(List<CreditWorthinessFactor> debts) {
        this.debts = debts;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/loan/CreditWorthinessSnapshot.kt
package org.apache.fineract.data.models.loan

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

import java.util.ArrayList

/**
 * @author Rajan Maurya
 * On 12/07/17.
 */

@Parcelize
data class CreditWorthinessSnapshot(
    @SerializedName("forCustomer") var forCustomer: String? = null,
    @SerializedName("incomeSources") var incomeSources: List<CreditWorthinessFactor> =
            ArrayList(),
    @SerializedName("assets") var assets: List<CreditWorthinessFactor> = ArrayList(),
    @SerializedName("debts") var debts: List<CreditWorthinessFactor> = ArrayList()
) : Parcelable